src/test/javascript/integration/webhook_localhost_spec.js

```diff
@@ -18,14 +18,17 @@ var webhookResource = utils.getWebhookUrl() + "/" + webhookName;
 
 describe(__filename, function () {
 
-    let isClustered = false;
+    let isClustered = true;
 
     it('determines if this is a single or clustered hub', (done) => {
         let url = `${hubUrlBase}/internal/properties`;
         utils.httpGet(url)
             .then(response => {
                 expect(response.statusCode).toEqual(200);
-                isClustered = response.body.properties['hub.type'] == 'aws';
+                let hubType = response.body.properties['hub.type'];
+                if (hubType !== undefined) {
+                    isClustered = response.body.properties['hub.type'] == 'aws';
```

Comment:
Should this be === ?

Reply:
In this case it doesn't matter since they're both strings. However it should be comparing ```hubType``` and not querying the response.body object again.

Commit incoming :)